esi_leap/objects/offer.py

```diff
@@ -58,6 +66,29 @@ def get_all_by_status(cls, context, status):
             context, status)
         return cls._from_db_object_list(context, db_offers)
 
+    @classmethod
```

Comment:
This doesn't need to be a class method; you can imagine it being called as

```
o = Offer.create(context, <values>)
o.send_to_flocx_market(context)
```

Also maybe call it `send_to_flocx_market` : )